e2e/src/main/protobuf/google/protobuf/fromstruct_two_level_nesting.proto

```diff
@@ -0,0 +1,56 @@
+// Protocol Buffers - Google's data interchange format
```

Comment:
Looks like this is an original file - so the copyright header is unnecessary.

Reply:
👍 would you like me to wait until you're done with the review before fixing?